e2e-tests/IIoTPlatform-E2E-Tests/TestConstants.cs

```diff
@@ -139,6 +139,31 @@ internal static class EnvironmentVariablesNames {
             /// IoT Hub connection string
             /// </summary>
             public const string PCS_IOTHUB_CONNSTRING = "PCS_IOTHUB_CONNSTRING";
+
+            /// <summary>
+            /// The connection string of the event-hub compatible endpoint of IoT Hub.
+            /// </summary>
+            public const string IOTHUB_EVENTHUB_CONNECTIONSTRING = "IOTHUB_EVENTHUB_CONNECTIONSTRING";
```

Comment:
We have `PCS_IOTHUB_EVENTHUBENDPOINT` for the same value. Maybe we can reuse that.